src/edfi-performance-test/edfi_performance_test/performance_tester.py

```diff
@@ -49,7 +49,7 @@ def spawn_pref_tests(args: MainArguments, user_class: Type[User]) -> None:
 
     # These tests expect that the base URL does NOT end with a slash
     if args.baseUrl.endswith("/"):
-        args.baseUrl = args.baseUrl[1:]
+        args.baseUrl = args.baseUrl[0:(len(args.baseUrl)-1)]
```

Comment:
That refactor I did last time... I thought I had tested it but ended up having something backwards here:  removed the first character instead of the last. That would be appropriate for the _resource_ (remove a starting slash), but not appropriate for the base url.